### ISSUE TITLE
Adding new people in a classifier

### ISSUE THREAD

**Issue by Liuxi-G** [ageitgey/face_recognition#493]
* face_recognition version:
* Python version:3.5
* Operating System:windows

### Description

Hi!
I'm right now using your method (KNN) to recognise people in a large dataset, my problem is if I have already trained a classieier (for example Facescrub), how can I add new people to recognise without training the part that has been trained already?
Thank you !
### What I Did

```
Paste the command(s) you ran and the output.
If there was a crash, please include the traceback here.
```

**Comment by Liuxi-G** [comment]
I'm trying to recognize as much face as I could in a large database

**Comment by cftang0827** [comment]
I don't think you need a knn classifier, you can just use L-2 norm distance to justify whether two faces are the same.

Besides, if you use KNN method, you need to retrain the classifier model as long as you add a new person.

Thanks

**Comment by xSNYPSx** [comment]
https://github.com/ageitgey/face_recognition/issues/787
